lib/ffprober/parsers/file.rb

```diff
@@ -0,0 +1,35 @@
+module Ffprober
+  module Parsers
+    class File
+
+      def initialize(file_to_parse)
+        unless ::File.exist?(file_to_parse)
+          fail ArgumentError.new("File not found #{file_to_parse}")
+        end
+
+        @file_to_parse = file_to_parse
+      end
+
+      def load
+        json_output = `#{ffprobe_finder.path} #{options} '#{@file_to_parse}'`
+        Ffprober::Parsers::Json.new(json_output)
+      end
+
+      private
+
+      def options
+        options = '-v quiet -print_format json -show_format -show_streams'
+        options << ' -show_chapters' if ffprobe_version.version >= Gem::Version.new('2.0.0')
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.